File: Laminatoria/DTO/OrderItemRequest.cs
using Laminatoria.Models;

namespace Laminatoria.DTO
{
    public class OrderItemRequest
    {
        public int Id { get; set; }
        public int Amount { get; set; }
        public string AdditionalPropValue { get; set; }
        public int ProductId { get; set; }        
        public int OrderId { get; set; }
        public decimal SumPrice { get; set; }
       
    }
}

File: laminatoria-client/src/app/classes/orderItemRequest.ts
export class OrderItemRequest {
	id: number
	amount: number
	additionalPropValue: string
	orderId: number
	productId: number
	sumPrice: number

	constructor(
		id: number,
		amount: number,
		additionalPropValue: string,
		orderId: number,
		productId: number,
		sumPrice: number
	) {
		this.id = id
		this.amount = amount
		this.additionalPropValue = additionalPropValue
		this.orderId = orderId
		this.productId = productId
		this.sumPrice = sumPrice
	}
}
